app.py

```diff
@@ -59,6 +59,26 @@ def node_register():
     return jsonify({})
 
 
+@app.route('/v1/proposals', methods=['GET'])
+def proposals():
+    node_key = request.args.get('node_key')
```

Comment:
Is it hard to make `node_key` optional filter?

Reply:
It should be easy, I'll add it.

Reply:
Done.